CONTRIBUTING.md

```diff
@@ -37,7 +37,7 @@ run the following command in the project main directory prior to finishing a PR:
 If encountering CI errors in CircleCI, it may be helpful to execute the test jobs
 locally to debug. This can be facilitated by executing the command
 
-   circleci local execute --job some-test-job
+   circleci local execute some-test-job [--job]
```

Comment:
This isn't logical. Only `circleci local execute [--job] some-test-job` is proper EBNF.

Reply:
Fixed in latest commit. Thanks @baentsch